src/parser/chord_grammar.pegjs

```diff
@@ -0,0 +1,23 @@
+Chord
```

Comment:
It is, it would be nicer if the grammar supported import directives.

We can, however, save the combined grammar as a git-ignored file to fix IDE inspections.